ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -17,10 +17,10 @@ spec:
   mcp: "master"
   sourceFiles:
     - fileName: ConsoleOperatorDisable.yaml
-      policyName: "console-policy"
+      policyName: "config-policy"
```

Comment:
Why is this a group vs common policy ?

Reply:
We talked about making it common but the point was raised that in a deployment with a mix of SNO and standard clusters they may want the console for standard(?). In that case a separate group could/would be made for group-du-std which doesn't contain the Console disable.

Reply:
Fair point